concepts/0020-message-types/README.md

```diff
@@ -61,7 +61,7 @@ did:sov:123456789abcdefghi1234;spec/exampleprotocol/1.0/exampletype
 
 This example DID document shows a service endpoint that includes a name property (emphasized) whose purpose is to enable creation of DID references that can deterministically select that service in order to have an algorithmic transformation into a concrete URI.
 
-```json
+```jsonc
 {
   "@context": "https://w3id.org/did/v1",
   "id": "did:example:123456789abcdefghi",
```

Comment:
Could yo change this as well? An ID should (in general) be a GUID, not a DID.

Reply:
@swcurran , this is an `id` in a DID doc, not an `id` in a DIDComm message. Therefore, having the `id` be a DID is actually normal.

Reply:
Doh.  Sorry.  Trying to figure out why a DIDDoc would be in that document.  It will be deleted soon :-)